bigbluebutton-html5/imports/ui/components/nav-bar/container.jsx

```diff
@@ -18,8 +19,17 @@ class NavBarContainer extends Component {
 }
 
 export default createContainer(() => {
+  let meetingTitle = Meetings.find().map(meeting => {
+    return meeting.meetingName;
```

Comment:
You could use shorthand syntax.
`let meetingTitle = Meetings.find().map(meeting => meeting.meetingName);`